src/test/java/nl/tudelft/ti2206/group9/gui/EndToEndTest.java

```diff
@@ -203,15 +205,17 @@ private void goThroughShop() {
         State.setCoins(COINS); //Make sure player has enough coins
         mainMenu(MAIN_SHOP);
 
-        assertEquals(State.getSkin(), Skin.getNoob());
+        final ObservableList<Skin> skinList = Skin.loadSkinsToList();
+
+        assertEquals(State.getSkin(), skinList.get(0));
         shopBuyEquipSkin(SHOP_SKIN_IRONMAN);
-        assertEquals(State.getSkin(), Skin.getNoob());
+        assertEquals(State.getSkin(), skinList.get(0));
         shopBuyEquipSkin(SHOP_SKIN_CAPTAIN);
-        assertEquals(State.getSkin(), Skin.getNoob());
+        assertEquals(State.getSkin(), skinList.get(0));
         shopBuyEquipSkin(SHOP_SKIN_CAPTAIN);
-        assertEquals(State.getSkin(), Skin.getAndy());
+        assertEquals(State.getSkin(), skinList.get(1));
         shopBuyEquipSkin(SHOP_SKIN_IRONMAN);
-        assertEquals(State.getSkin(), Skin.getNoob());
+        assertEquals(State.getSkin(), skinList.get(0));
```

Comment:
Well, the buttons that get clicked in the interleaved lines also have hardcoded numbers 0 and 1. The E2E doesn't have to care which skins are actually bought, as long as it notices "when I click on button no. 1, skin no. 1 is bought. When I click again, skin no. 1 is equipped". Positions 0 and 1 (maybe even 2) can be used for this purpose.

<review, testing, logic